app/views/pages/amends/outcomeActive.scala.html

```diff
@@ -62,9 +62,13 @@ <h2>@Messages("pla.resultSuccess.IPChangeDetails")</h2>
 
     <h2>@Messages("pla.resultSuccess.giveFeedback")</h2>
 
-    <p><a href=@controllers.routes.ExitSurveyController.exitSurvey>@Messages("pla.resultSuccess.exitSurveyLinkText")</a> @Messages("pla.resultSuccess.exitSurvey")</p>
-
+    <p>
+        <a id="submit-survey-button" href=@controllers.routes.AccountController.signOut class="button button-start" role="button" onclick="submitExitSurvey()">
```

Comment:
This applies to the other changes similar to this one. You've replaced the content here from text to a button. The button also doesn't seem to link me to the feedback survey?